app/controllers/admins_controller.rb

```diff
@@ -1,60 +1,42 @@
 class AdminsController < ApplicationController
-    before_action :authenticate_user!
+  before_action :authenticate_user!
     
-    def index
-      @items=Item.all
-      @procedures=Procedure.all
-      @user=current_user
```

Comment:
[Style] (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)
These three statements all need spaces around the equal sign.